#! /bin/sh /usr/share/dpatch/dpatch-run
## use-nusoap-from-distro.dpatch by Olivier Berger <olivier.berger@it-sudparis.eu>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Use the Debian nusoap from libusoap-php

@DPATCH@

=== modified file 'gforge/www/soap/index.php'
--- gforge/www/soap/index.php	2009-04-03 14:47:39 +0000
+++ gforge/www/soap/index.php	2009-11-14 08:25:56 +0000
@@ -11,7 +11,8 @@
 
 $uri = 'http://'.$sys_default_domain;
 // 1. include client and server
-require_once $gfwww.'soap/nusoap.php';
+#require_once $gfwww.'soap/nusoap.php';
+require_once '/usr/share/php/nusoap/nusoap.php';
 //$debug = true;
 // 2. instantiate server object
 $server = new soap_server();

